gradle-hsql-plugin/src/main/groovy/HsqldbStopTask.groovy

```diff
@@ -0,0 +1,32 @@
+package pl.tlempart.gradle.plugins.hsqldb
```

Comment:
This `HsqldbStopTask.groovy` looks similar to [Tomasz Lempart's](https://github.com/tlempart/gradle-hsqldb-plugin/blob/master/gradle-hsqldb-plugin/src/main/groovy/pl/tlempart/gradle/plugins/hsqldb/HsqldbStopTask.groovy). I don't understand whether and what open source license they might be making this software available under. 

Incorporating third-party code has implications for required acknowledgement in the source code, in the `NOTICES` file. But without clarity on what licensing basis it's permissible for uPortal to redistribute this code, uPortal may not be able to use this at all.